.changeset/good-drinks-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/dashboard': minor
+---
+
+feat: project health
```

Comment:
we might need to elaborate more on this: something like

`feat: add services health checks indicators to the overview page`

Reply:
updated in last commit